.gitignore

```diff
@@ -4,6 +4,13 @@ artifacts/
 cache/
 dist/
 .secret
+.env
+
+test/inputs/
```

Comment:
Those paths should be generated under TEST_PATH, and recommend setting the TEST_PATH out of the project workspace.